docs/specifications/tests/Basic-TP/basic02.md

```diff
@@ -104,38 +104,26 @@ queries follow, unless otherwise specified below, what is specified for
    2. Exit these test procedures.
 
 5. Else, for each name server name in the *Delegation NS* set do:
-   1. If the name server name has no IP address then do:
-      1. Add the name server name to the *NS Cannot Resolve Into IP*.
-      2. Go to next name server name.
-   2. If the name server name has multiple IP addresses, then repeat the loop
-      and the following steps for each IP address, else go to next name server
-      name in the loop.
-   3. Send *SOA Query* to the name server IP.
-   4. If there is no [DNS Response], then:
-      1. Add the name server name and IP address to the *No Response From NS*
-         set.
-      2. Go to next name server.
-   5. Else, if the AA flag is not set in the [DNS Response], then:
-      1. Add the name server name and IP address to the *NS not auth* set.
-      2. Go to next name server.
-   6. Else, if the [RCODE Name] is not "NoError" in the [DNS Response], then:
-      1. Add the name server name and IP address plus the [RCODE Name] to the
+   1. If the name server name has no IP address then add the name server name to
+      the *NS Cannot Resolve Into IP* set.
+   2. Else, for each IP address in for the name server name do:
```

Comment:
missing word after "in" (maybe "in the IP addess set"?)

Reply:
Fixed.